cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -176,4 +194,8 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         assessmentApplication.perform_assessment("low", [stakeHolder]);
         assessmentApplication.verifyStatus("assessment", "Completed");
     });
+
+    it("Assess archetype", function () {
```

Comment:
Suggestion:
Archetype creation and assessment could be combined into a single test , just like application creation and assesment have been combined into a single test.